Assets/_BForBoss/_Character/Scripts/PlayerDashBehaviour.cs

```diff
@@ -36,13 +34,6 @@ public void Initialize(ECM2.Characters.Character baseCharacter, Func<Vector2> ch
             _characterInputMovement = characterMovement;
             _onDashing = onDashing;
         }
-        
-        public void SetupPlayerInput(InputAction dashAction)
```

Comment:
Removed the `InputAction` from dash, and handle everything in `PlayerMovement` script